app/src/main/java/org/schabi/newpipe/info_list/StreamSegmentItem.kt

```diff
@@ -29,6 +30,14 @@ class StreamSegmentItem(
             )
         }
         viewHolder.root.findViewById<TextView>(R.id.textViewTitle).text = item.title
+        if (item.channelName == null) {
+            viewHolder.root.findViewById<TextView>(R.id.textViewChannel).visibility = View.GONE
```

Comment:
It could happen that only some items in the list set a channel name, therefore the visibility should be set to `View.VISIBLE` in the `else` block.